<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>candatacapture</class>
 <widget class="QMainWindow" name="candatacapture">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1126</width>
    <height>873</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTableWidget" name="tableWidget_show">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>180</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>180</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(235, 235, 235);</string>
      </property>
      <widget class="QWidget" name="widget_2" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>201</width>
         <height>180</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>180</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>180</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>IP</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_serverIP">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>140</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="placeholderText">
             <string>192.168.225.1</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>PORT</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_serverPort">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>140</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string>16002~4</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="pushButton_disconnect">
            <property name="text">
             <string>断开</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_connect">
            <property name="text">
             <string>连接</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QTableWidget" name="tableWidget_input">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>0</y>
         <width>420</width>
         <height>180</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>420</width>
         <height>180</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>180</height>
        </size>
       </property>
      </widget>
      <widget class="QWidget" name="widget_3" native="true">
       <property name="geometry">
        <rect>
         <x>620</x>
         <y>0</y>
         <width>230</width>
         <height>180</height>
        </rect>
       </property>
       <widget class="QLabel" name="label_sendcounts">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>72</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>发送次数:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_sendCounts">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>30</y>
          <width>140</width>
          <height>21</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>21</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>21</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>1</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_sendInterval">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>每次发送间隔(ms):</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_sendFrameInterval">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>每帧发送间隔(ms):</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_sendinterval">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>70</y>
          <width>80</width>
          <height>21</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>21</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>21</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>1000</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_sendFrameInterval">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>100</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>100</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_sendFrame">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>150</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>发送</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_stopFrame">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>150</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>停止</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1126</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
